pages/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+
+const Lista = (props) => (
+  <div>
+    <ul>
+      {props.ideas.map(function(idea, index){
```

Comment:
aprovechá la sintaxis de ES6 de arrow functions (como hacés más arriba):

{ props.ideas.map((idea, index) => <li key={ index }>{ idea }</li>) }

notá que fui consistente con el tema de spacing y `{ }`